Source/VMware.vSphereDSC/DSCResources/VDPortGroup.ps1

```diff
@@ -273,10 +334,40 @@ class VDPortGroup : BaseDSC {
         $distributedPortGroupParams = $this.GetDistributedPortGroupParams()
 
         try {
+            Write-VerboseLog -Message $this.ModifyVDPortGroupMessage -Arguments @($distributedPortGroup.Name)
```

Comment:
Here as well **Write-VerboseLog** will need to be replaced by **WriteLogUtil** after the other PR gets approved.